scripts/firedrake-install

```diff
@@ -111,6 +112,8 @@ else:
                         help="Rebuild all packages even if no new version is available. Usually petsc and petsc4py are only rebuilt if they change. All other packages are always rebuilt.")
     parser.add_argument("--adjoint", action='store_true', dest="install_adjoint",
                         help="Also install firedrake-adjoint.")
+    parser.add_argument("--slepc", action="store_true",
+                        help="Install SLEPc along with PETSc")
```

Comment:
This is not the right way to do this. You need to copy what happens for --adjoint